### PR TITLE
Search text changed on search by req

### PR DESCRIPTION
**Pull request by adeniji19**
I think the best choice is to put generic text so the user knows what is being displayed but doesn't think they can replicate by typing it into the search bar. 

Resolves: #125

### CODE REVIEW

**Review by churichard — CHANGES_REQUESTED** [review]
I think displaying the requirement title would be more informative, since the user might forget what was searched.

**Comment by bnehoran** [comment]
I removed the commits that were unrelated to this pull request, so if your local branch still has those commits but you want to pull, you just have to do `git reset --hard master` before doing `git pull`.

----------------------------
Caveat: If you have local, unpushed commits you want to keep, you might want to first make a copy of the branch. For instance,
```bash
git branch old_search_text
git reset --hard master
git pull
git cherry-pick old_search_text~[n]..old_search_text #replace the [n] with the number of commits to keep
```

**Comment by bnehoran** [comment]
I removed the commits that were unrelated to this pull request and force pushed, so if your local branch still has those commits but you want to pull, you just have to do `git reset --hard master` before doing `git pull`.

----------------------------
Caveat: If you have local, unpushed commits you want to keep, you might want to first make a copy of the local branch. For instance,
```bash
git branch old_search_text
git reset --hard master
git pull
git cherry-pick old_search_text~[n]..old_search_text #replace the [n] with the number of commits to keep
```

**Review by bnehoran — APPROVED** [review]

**Comment by bnehoran** [comment]
Resolves: #125

File: frontend/src/Requirements.js [hunk 1]
```diff
@@ -36,6 +36,8 @@ function getHash(stringName) {
 }
 
 function getReqCourses(req_path){
+  var filterText = 'Satisfying: ' + req_path.substring(req_path.lastIndexOf("//")+2,req_path.length);
```
Comment (churichard) [inline frontend/src/Requirements.js]: A cleaner way to do this is by doing `req_path.split('//').pop()`

File: frontend/src/Search.js [hunk 2]
```diff
@@ -309,6 +309,7 @@ class Search extends Component {
     return (
         <div>
         <input type = "text" 
+          name="search-text"
```
Comment (churichard) [inline frontend/src/Search.js]: Don't use `name`, that's only used in forms. If you want to identify an element, use the field `id`

File: frontend/src/Requirements.js [hunk 3]
```diff
@@ -36,6 +36,8 @@ function getHash(stringName) {
...
+  document.getElementsByName('search-text')[0].value=filterText;
```
Comment (churichard) [inline frontend/src/Requirements.js]: jQuery can do this more cleanly: `$('#search-text').val(filterText);`

**Review by churichard — CHANGES_REQUESTED** [review]

File: frontend/src/Search.js [hunk 4]
```diff
@@ -305,10 +305,11 @@ class Search extends Component {
 
   }
   render()
-  { 
+  {
     return (
         <div>
-        <input type = "text" 
+        <input type = "text"
+          input id="search-text"
```
Comment (churichard) [inline frontend/src/Search.js]: Don't need `input` here, just `id` is fine.

**Review by churichard — CHANGES_REQUESTED** [review]

**Review by churichard — APPROVED** [review]

**Pull request merged**